mima.sbt

```diff
@@ -145,6 +145,10 @@ ThisBuild / mimaBinaryIssueFilters ++= {
       exclude[ReversedAbstractMethodProblem]("cats.free.ContravariantCoyoneda.k"),
       exclude[DirectAbstractMethodProblem]("cats.free.Coyoneda.k"),
       exclude[ReversedAbstractMethodProblem]("cats.free.Coyoneda.k")
+    ) ++ // https://typelevel/cats/pull/2607
+    Seq(
+      exclude[DirectMissingMethodProblem]("cats.instances.EqInstances.$init$"),
+      exclude[DirectMissingMethodProblem]("cats.instances.EquivInstances.$init$")
     )
```

Comment:
So these are actually legit bincompatibility breaks, that we ideally should not exclude. Although Scala 3 only. I am thinking about how we might best workaround these. The issue is that we removed the `val` from `EqInstances`.

Reply:
Proposed a solution in https://github.com/typelevel/cats/pull/2607#pullrequestreview-1026674706.

Reply:
Yeah, I'm afraid I rely on the guidance of the maintainers on these - I simply don't work with bincompat enough to have any intuition.

Reply:
No worries! Bincompat can be very counter-intuitive even for experienced maintainers. I'm just glad we could fix this one, thanks!